python/ql/test/experimental/dataflow/ApiGraphs/test_import_star.py

```diff
@@ -0,0 +1,23 @@
+# Star imports
+
+from unknown import * #$ use=moduleImport("unknown")
+
+# Currently missing, as we do not consider `hello` to be a `LocalSourceNode`, since it has flow
+# going into it from its corresponding `GlobalSsaVariable`.
+hello() #$ MISSING: use=moduleImport("unknown").getMember("hello").getReturn()
+
+non_module_member
+
+outer_bar = 5
+outer_bar
```

Comment:
It took me a while to figure out what these are about. 

For `non_module_member` I don't really understand it. Since the `quux` function couldn't have run yet, it seems like it will give just give `NameError`. But I guess the intention is to check whether the analysis would think that it could come from the `unknown` module?

And same goes for `outer_bar`?

If that's the case, how about adding a line explaining this?

```suggestion
# We don't want our analysis to think that either `non_module_member` or `outer_bar` can
# come from `from unknown import *`
non_module_member

outer_bar = 5
outer_bar
```

Reply:
Yeah, that's exactly right. Because these kinds of imports are so polluting, I wanted to make sure that our modelling was as tight as reasonably possible, hence the many other kinds of unknown variables. An explanatory comment is a good idea. :+1: